lasagne/layers/sparse/dense.py

```diff
@@ -0,0 +1,24 @@
+import theano.sparse as sparse
+
+from ..dense import DenseLayer as Dense
+
+
+__all__ = [
+    "DenseLayer",
+]
+
+
+class DenseLayer(Dense):
+    def get_output_for(self, input, **kwargs):
+        """
+        Parameters
+        ----------
+        input : theano sparse matrix
+            output from the previous layer
+        """
+        assert type(input) == sparse.basic.SparseVariable
+
+        activation = sparse.basic.dot(input, self.W)
```

Comment:
Right, it don't make a difference in this case. So if you wish, with theano.dot(), you could make the dense layer work with both Type.